### DISCUSSION TITLE
is it possible to ignore just the sub-stream in the web UI?

### DISCUSSION THREAD

**Discussion by cheme75** [mrlt8/docker-wyze-bridge#800] [Q&A] [answered]
just wondering if there is an option to ignore or hide the sub-stream from displaying in the web ui? I appreciate having the sub-stream for scrypted to use for motion detection, but now I have 2 streams in the web ui for a couple of cams which I do not need to view, I really just want to display and view the primary streams. If there is an option, I've missed it - been a bit diverted with a long over due family visit, but I'm back now and hope to have more time for this.

**Comment by mrlt8** [comment, marked as answer]
Not right now, but it is something that is on the todo list.
